api/src/main/java/module-info.java

```diff
@@ -172,13 +177,14 @@
  *
  * <h2>Methods with Entity Parameters</h2>
  *
- * <p>You can annotate a method with {@link Insert}, {@link Update}, {@link Save},
- * or {@link Delete} if the method accepts a single parameter, which must be one of:</p>
+ * <p>A method may be annotated with a lifecycle annotation such as {@link Insert},
+ * {@link Update}, {@link Save}, or {@link Delete}. The method must accept a single
+ * parameter, whose type is either:</p>
  *
  * <ul>
- * <li>An entity.</li>
- * <li>An array of entity (variable arguments array is permitted).</li>
- * <li>An {@link Iterable} of entity (subclasses such as {@link List} are permitted).</li>
+ * <li>the class of the entity to be updated, or</li>
+ * <li>{@code Iterable<E>} or {@code E[]} where {@code E} is the class of the
+ *     entities to be updated.</li>
```

Comment:
This applies to delete in addition to update/save
```suggestion
 * <li>the class of the entity, or</li>
 * <li>{@code Iterable<E>} or {@code E[]} where {@code E} is the class of the
 *     entities.</li>
```